From e9cbca077f9b9501e354cc980f337a34c33847f4 Mon Sep 17 00:00:00 2001
From: Jonathan Hsieh <jmhsieh@apache.org>
Date: Tue, 6 Sep 2011 00:49:37 +0000
Subject: [PATCH 42/65] FLUME-758: Test Excludes on maven build doesn't always work

Surefire 2.8.1 does not allow for lists of excluded tests to be specified via command line.
We force the build to use surefire 2.6 which does.

Verions 2.9, 2.8, 2.7.2, 2.7.1 and 2.7 were also tested and they failed to exclude.

git-svn-id: https://svn.apache.org/repos/asf/incubator/flume/trunk@1165464 13f79535-47bb-0310-9956-ffa450edef68
---
 pom.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index e341607..6eb115c 100644
--- a/pom.xml
+++ b/pom.xml
@@ -380,7 +380,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.8.1</version>
+          <version>2.6</version>
           <configuration>
             <forkMode>always</forkMode>
             <forkedProcessTimeoutInSeconds>480</forkedProcessTimeoutInSeconds>
-- 
1.7.0.4

